foam_wrapper/foam_wrapper.py

```diff
@@ -0,0 +1,158 @@
+""" foam_wrapper module
+
+Wrapper module for OpenFOAM using pythonFlu -python interface
+NOTE: at a moment pythonFlu supports only up to 2.0.1 -version of OpenFOAM
+
+"""
+
+from Foam import ref, man, src
+from simphony.cuds.mesh import Mesh, Point, Face, Edge, Cell
+import inspect
+from Foam.OpenFOAM import fileName, word, IOobject
+from collections import defaultdict
+from Foam.finiteVolume import volScalarField
+from Foam.src.OpenFOAM import *
+
+class FoamWrapper(object):
```

Comment:
On the wiki, we proposed an interface for wrapper to a modeling engine (see https://publicwiki-01.fraunhofer.de/SimPhoNy-Project/index.php/Wrapper_description and the bottom where there is a class diagram).   In the FoamWrapper, there is the `get_mesh` method which matches up well.  Lets either use the wiki (or github) to keep the modeling-engine wrappers with the same interface.